### PR TITLE
Update charter.txt

### PR DESCRIPTION
**Pull request by jankow72**

### CODE REVIEW

**Comment by jankow72** [comment]
I updated the roles to rotate at every meeting

File: charter.txt [hunk 1]
```diff
@@ -18,10 +18,14 @@ Properly tested, checked into Github, meets all requirements, checked by teammat
 
 Scrum meeting format/frequency and distribution of documentation tasks (meeting minutes, updating task board):
 ~15 minutes 
-Take notes - Amenah
-Update task board - Kaitlyn
-Facilitate questions - Megan
-Code Review - Cattleia
+
+Roles to switch weekly:
```
Comment (amenahs) [inline charter.txt:22]: Approved change

**Comment by mnkocher** [comment]
Looks good

**Review by amenahs — COMMENTED** [review]
great!

**Review by mnkocher — APPROVED** [review]
Looks good

**Pull request merged**